src/avfoundation.cs

```diff
@@ -1827,204 +1832,241 @@ interface AVAudioSessionSecondaryAudioHintEventArgs {
 
 	delegate void AVPermissionGranted (bool granted);
 
-	[NoMac]
+	[Mac (11,0)]
 	[Watch (3,0)]
+#if MONOMAC
+	[Static]
```

Comment:
What does this attribute do? IIRC it only make sense for members, not types.

Reply:
it works on types too, it makes a static type - which is how we regrouped constants (before smart enums)